src/test/scala/io/github/paoloboni/binance/FapiClientIntegrationTest.scala

```diff
@@ -504,21 +504,27 @@ class FapiClientIntegrationTest extends AnyFreeSpec with Matchers with EitherVal
 
     implicit val withClock: WithClock[IO] = WithClock.create(stubTimer(fixedTime))
 
+    import io.circe.syntax._
+    val b = FutureOrderCreateParams
+      .MARKET(
+        symbol = "BTCUSDT",
+        side = OrderSide.BUY,
+        positionSide = FuturePositionSide.BOTH,
+        quantity = 10
+      )
+      .asInstanceOf[FutureOrderCreateParams]
+    .asJson
+
+    println(b)
```

Comment:
remove?

Reply:
oh yes, i am sorry this is just my highly professional debugging :D